test/integration/compliance_history_test.go

```diff
@@ -614,6 +615,9 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
 		)
 		Expect(err).ToNot(HaveOccurred())
 
+		By("Verifying that the policy has created the namespace")
+		verifyPolicyOnAllClusters(ctx, policyNS, prereqPolicyName, "Compliant", defaultTimeoutSeconds)
```

Comment:
Well. This seems important. 😆

Reply:
Do you want to bump the timout to `defaultTimeoutSeconds *2`? Below as well?

Reply:
I expect it to be fine at 30 seconds. The config policy controller just needs to get to the policy once. If we have to use `default * 2` for everything, maybe we should just increase the default :laughing: